### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**

### CODE REVIEW

File: ip-quality-score/info.json [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "server_url",
+        "value": "https://ipqualityscore.com",
+        "description": "The URL of the IPQualityScore server to which you will connect and perform the automated operations."
+      },
+      {
+        "title": "Private Kay",
```
Comment (cs-prerna-joshi) [inline ip-quality-score/info.json]:
```suggestion
        "title": "Private Key",
```

File: ip-quality-score/playbooks/playbooks.json [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+  "type": "workflow_collections",
+  "data": [
+    {
+      "uuid": "fafabd46-997d-41be-92a2-26855925a435",
+      "@type": "WorkflowCollection",
+      "name": "Sample - IP Quality Score - 1.0.0",
+      "description": "The IPQualityScore (IPQS) Threat Intelligence application provides threat intelligence for IP addresses, email addresses, URLs, and domains. This connector facilitates automated interactions with a IP Quality Score server using FortiSOAR\u2122 playbooks.",
+      "visible": true,
+      "image": null,
+      "recordTags": [
```
Comment (cs-prerna-joshi) [inline ip-quality-score/playbooks/playbooks.json:11]: mentioned the tags.

File: ip-quality-score/info.json [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "options": [
+            "0",
+            "1",
+            "2"
+          ],
+          "value": [
+            "0"
```
Comment (cs-prerna-joshi) [inline ip-quality-score/info.json]: here api accept string or integer?

Comment (cs-sujeet-ahirrao) [reply ip-quality-score/info.json]: Modified the info.json and convert string to int

File: ip-quality-score/playbooks/playbooks.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply ip-quality-score/playbooks/playbooks.json:11]: done

File: ip-quality-score/info.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply ip-quality-score/info.json]: done

**Review by cs-prerna-joshi — APPROVED** [review]
LGTM

**Pull request merged**